common/micc.js

```diff
@@ -69,6 +85,65 @@ function Micc(serverAddress) {
         this.postRequest(`openmedia`, body, processResponse);
     }
 
+    this.putQueueConversation = function (queueId, conversationId, body, processResponse) {
+        if (!queueId) {
+            console.log('Will not perform action because no queue id given.');
+        } else if (!conversationId) {
+            console.log('Will not perform action because no conversation id given.');
+        } else {
+            this.putRequest(`queues/${queueId}/conversations/${conversationId}`, body, processResponse);
+        }
+    }
+
+    this.pickQueueConversation = function (conversationId, queueId) {
+        this.putQueueConversation(queueId, conversationId, `{
+            conversationAction: 'Pick',
+            id: '${conversationId}'
+            }`, function processResponse(responseData) {
+                console.log(`Response for 'Pick' for conversation ${conversationId}:  `, responseData);
```

Comment:
Careful.  There may not be a body in the response but there's most certainly an HTTP status code.